src/pages/species/component.jsx

```diff
@@ -1,14 +1,159 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import './styles.scss';
+import { Link } from 'react-router-dom';
+import { useQuery } from 'urql';
+import findIndex from 'lodash/findIndex';
+import Dropdown from 'components/dropdown';
+import Dashboard from 'components/dashboard';
+import styles from './styles.scss';
+import SpeciesList from './components/species-list/component';
+import mockData from './data.json';
 
 function SpeciesPage({ match }) {
   const { iso, id } = (match && match.params) || {};
+  const wikiURL = 'https://en.wikipedia.org/api/rest_v1/page/summary/';
+  const [wikiInfo, setInfo] = useState('');
+  const [countries, setCountries] = useState([]);
+  const [activeCountry, setCountry] = useState(null);
+  const [species, setSpecies] = useState([]);
+  const [activeSpecie, setSpecie] = useState(null);
+
+  const [result] = useQuery({
+    query: `
+    {
+      countrySpecieDistributions(where:{
+        country: {
+          iso: "${iso}"
+        }
+      }) {
+        country {
+          iso,
+          name
+        },
+        specie {
+          name,
+          scientificName,
+          wikipediaSlug,
+          id
+        }
+      },
+      countries {
+        iso,
+        name
+      }
+    }
+    `
+  });
+
+  const { fetching, data, error } = result;
+
+  useEffect(() => {
+    if (data) {
+      const distrib = data.countrySpecieDistributions;
+      const countryList = data.countries;
+      setCountry(distrib && distrib[0] && distrib[0].country);
```

Comment:
I think this should be a memo variable not a state. This is derived from the url. We can talk about it if you want.